README.md

```diff
@@ -157,32 +157,28 @@ Though it would never be displayed as part of an aggregated feed unless requeste
 This is how it might be stored in a database system:
 
     "accessControls":	{
-    	"access":   {
-    		"keys":		[
-    			"secretkey123",
-    			"secretkey456"	
-    		],
-    		"users":	[
-    			"root",
-    			"john"
-    		],
-    		"roles":	[
-    			"employee"
-    		]
-    	},
-    	"admin": {
-    		"keys":		[
-    			"supersecretkey12345"
-    		],
-    		"users":	[
-    			"root"
-    		],
-    		"roles":    [
-    			"manager"
-    		]
-    	}
+        "users": {
+            "john": [
+            	"read"
+            ],
+            "fred": [
+                { "read": "deny" }
+            ]
+        },
+        "roles": {
+            "staff": [
+                "read"
+            ],
+            "admin": [
+                "write"
+            ]
+        }
```

Comment:
Good point.  An object should be sufficient, cuz it's not like we're assigning read:true and read:false to the same user.